src/pages/documentation/platform-assurance/tutorials/validation-summary.md

```diff
@@ -36,7 +31,7 @@ After you have tested your function, you can select **Upload** to save your vali
 | `uuid` | String | Universally unique identifier for the event. |
 | `timestamp` | Number | Timestamp from the client when the event was sent to Assurance. |
 | `eventNumber` | Number | Used to order when the event was sent. This key is useful when events have the same timestamp. |
-| `vendor` | String | Vendor identification string in the reverse domain name format (for example, com.adobe.assurance). |
```

Comment:
formatting and renaming - can we revert this change to remove the backslashes and use the new name?